### PR TITLE
Home Page & Global Style Updates

### PR DESCRIPTION
**Pull request by SolivanLau**
**Media Query Issues**
- have been having some mq issues with variables. not sure why vars do not work with mq. static values in for now
- unsure if multiple css files or live server messing with some mqs. have been having some issues loading mq in browser

**Design Deviation**
- footer links is simplified. contact link is removed from policy. now in footer nav
- need some time to figure out responsive hero image. 

**Responsive Styles**
- working down to 320px!
- I have moved over as many styles from **home.css**  that I believe are applicable to **global.css**. please check when you are coding through other pages. 
- I have added **visibility: hidden** + **display: none** to remove unwanted elements from the Accessibility DOM.  
- still need to update srcset for optimized pictures!

**Modal Work**
- working and responsive! 
- Has some global classes to use for other modals (bios in about us) that we can reuse later for other modals

### CODE REVIEW

File: css/global.css [hunk 1]
```diff
@@ -724,9 +808,78 @@ Feel free to change the min/max-width values to whatever you need. Use variables
 }
 
 /* Tablet and smaller 768px */
-@media screen and (max-width: var(--mq-width-tablet-max)) {
+@media screen and (max-width: 768px) {
```
Comment (SolivanLau) [inline css/global.css:816]: not sure why css vars not working with media query.  I have been trying to use --mq-width-tablet-max, but it does not register during tests.

I left in static values for now 🤔

File: css/global.css [hunk 2]
```diff
@@ -724,9 +808,78 @@ Feel free to change the min/max-width values to whatever you need. Use variables
...
+	===================================
+	*/
+	.modal-content {
+		width: 90%;
+		padding: var(--spacing-sm);
+	}
 }
 
+/* custom breakpoint 470px */
+@media screen and (max-width: 470px) {
```
Comment (SolivanLau) [inline css/global.css:851]: I have been trying to use --mq-width-phone-max, but it does not register during tests. 470px is working well as a  mobile breakpoint, let's consider 470px vs. 400px

File: css/home.css [hunk 3]
```diff
@@ -102,53 +96,407 @@ Community Partnerships
...
 ===================================
 Media Queries (overriding styles for different screen sizes)
+===================================
+*/
 
-Note: Here we are using a mobile-first approach, so we start with the smallest screen size and work our way up.
-Start by writing styles for the mobile version of the design on the lines above.
-Anything that needs to change for larger screens should be added to the media queries below.
+/* Tablet and smaller 1024px*/
+@media screen and (max-width: 1024px) {
```
Comment (SolivanLau) [inline css/home.css:310]: More static values for media query work - will do some research why these vars are not working well!

File: index.html [hunk 4]
```diff
@@ -119,7 +122,10 @@ <h2>Preserving Tomorrow by Building Today</h2>
 					<div class="community-card">
 						<!-- img -->
 						<div class="img-container">
-							<img src="https://placedog.net/800/640?id=7" alt="doggo" />
+							<img
+								src="./assets/images/pages/home/community-partnership.jpg"
```
Comment (SolivanLau) [inline index.html:131]: @maxharrisnet can you please crop and reoptimize this picture? it has a rounded corner baked into the file.

**Review by maxharrisnet — APPROVED** [review]

**Pull request merged**